modules/ROOT/pages/ai-bedrock.adoc

```diff
@@ -0,0 +1,209 @@
+= Amazon Bedrock integration guide
+:pluginname: AI Assistant
+:plugincode: ai
+:plugincategory: premium
+:navtitle: Amazon Bedrock integration guide
+:description_short: {pluginname} with Amazon Bedrock
+:description: A guide for integrating {pluginname} plugin using Amazon Bedrock.
+:keywords: example, demo, custom, plugin, ai, assistant, guide, amazon, bedrock, aws
+
+include::partial$misc/admon-ai-pricing.adoc[]
+
+== Introduction
+
+This guide provides instructions for integrating the {pluginname} plugin using https://docs.aws.amazon.com/bedrock/latest/userguide/setting-up.html[Amazon Bedrock] in {productname}. Amazon Bedrock is a managed service for building generative AI applications on AWS. The advantage of using Amazon Bedrock is that it provides a wide range of foundation models that can be used interchangeably with little to no modification.
+
+The following examples load the AWS credentials directly on the client side. For security reasons, it is recommended to use an alternate method for retrieving these credentials. It is recommended to hide the API calls behind a server-side proxy to avoid exposing the AWS credentials to the client side.
+
+These examples use Node.js and https://docs.aws.amazon.com/sdk-for-javascript/v3/developer-guide/welcome.html[AWS SDK for JavaScript] through the `@aws-sdk/client-bedrock-runtime` package to interact with the Amazon Bedrock API. However, you can use any development environment that the https://aws.amazon.com/developer/tools[AWS SDKs] support.
+
+Here, the Anthropic Claude-3 Haiku model is used as an example. You can replace `modelId` with the model you want to use. See https://docs.aws.amazon.com/bedrock/latest/userguide/models-supported.html[Supported models] for more information. Note that each foundation model comes with its own set of parameters.
+
+To learn more about the difference between string and streaming responses, see xref:ai.adoc#the-respondwith-object[The `respondWith` object] on the plugin page.
+
+== Prerequisites
+
+Before you begin, you need the following:
+
+1. An AWS account with access to Amazon Bedrock.
+2. The AWS credentials for the account.
+3. A Node.js environment with the `@aws-sdk/client-bedrock-runtime` package installed.
```

Comment:
@FarzadHayat you could use standard asciidoc syntax here for numlists rather than writing them.
```suggestion
. An AWS account with access to Amazon Bedrock.
. The AWS credentials for the account.
. A Node.js environment with the `@aws-sdk/client-bedrock-runtime` package installed.
```